src/main/java/onlinebookstore/controller/BookController.java

```diff
@@ -32,14 +32,14 @@ public class BookController {
     private final BookService bookService;
 
     @GetMapping
-    @PreAuthorize("hasRole('ROLE_USER')")
+    @PreAuthorize("hasAnyRole('ROLE_USER', 'ROLE_ADMIN')")
```

Comment:
1. you may omit `ROLE_` prefix - it is included within `hasRole` as is
2. shouldn't admin user also have a role `USER`? or you design your app in a way that admin does not have same priviliges as user has?